{"sentences": "### Summary\n\nI was learning nextjs through the official documentation. At [this](https://nextjs.org/learn/dashboard-app/setting-up-your-database) section on the tutorial, when I tried to run `npm run seed` it throws me the following error. I tried running the same command by copying the files to windows `C:\\Users\\abhis\\Documents` and from this folder the command runs fine and no issues. what is wrong with running this on wsl?\r\n```\r\nnode:events:496\r\n      throw er; // Unhandled 'error' event\r\n      ^\r\n\r\nAggregateError [ETIMEDOUT]: \r\n    at internalConnectMultiple (node:net:1116:18)\r\n    at internalConnectMultiple (node:net:1184:5)\r\n    at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n    at listOnTimeout (node:internal/timers:575:11)\r\n    at process.processTimers (node:internal/timers:514:7)\r\nEmitted 'error' event on WebSocket instance at:\r\n    at emitErrorAndClose (/home/abishek/nextjs-dashboard/node_modules/ws/lib/websocket.js:1016:13)\r\n    at ClientRequest.<anonymous> (/home/abishek/nextjs-dashboard/node_modules/ws/lib/websocket.js:864:5)\r\n    at ClientRequest.emit (node:events:518:28)\r\n    at TLSSocket.socketErrorListener (node:_http_client:500:9)\r\n    at TLSSocket.emit (node:events:518:28)\r\n    at emitErrorNT (node:internal/streams/destroy:169:8)\r\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\r\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {\r\n  code: 'ETIMEDOUT',\r\n  [errors]: [\r\n    Error: connect ETIMEDOUT 34.194.100.28:443\r\n        at createConnectionError (node:net:1646:14)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1705:38)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -110,\r\n      code: 'ETIMEDOUT',\r\n      syscall: 'connect',\r\n      address: '34.194.100.28',\r\n      port: 443\r\n    },\r\n    Error: connect ENETUNREACH 2600:1f18:240c:5421:acd1:6284:5a8a:d6e9:443 - Local (:::0)\r\n        at internalConnectMultiple (node:net:1180:16)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -101,\r\n      code: 'ENETUNREACH',\r\n      syscall: 'connect',\r\n      address: '2600:1f18:240c:5421:acd1:6284:5a8a:d6e9',\r\n      port: 443\r\n    },\r\n    Error: connect ETIMEDOUT 44.207.148.149:443\r\n        at createConnectionError (node:net:1646:14)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1705:38)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -110,\r\n      code: 'ETIMEDOUT',\r\n      syscall: 'connect',\r\n      address: '44.207.148.149',\r\n      port: 443\r\n    },\r\n    Error: connect ENETUNREACH 2600:1f18:240c:5454:82a4:8209:20f8:a5c1:443 - Local (:::0)\r\n        at internalConnectMultiple (node:net:1180:16)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -101,\r\n      code: 'ENETUNREACH',\r\n      syscall: 'connect',\r\n      address: '2600:1f18:240c:5454:82a4:8209:20f8:a5c1',\r\n      port: 443\r\n    },\r\n    Error: connect ETIMEDOUT 23.23.229.9:443\r\n        at createConnectionError (node:net:1646:14)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1705:38)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -110,\r\n      code: 'ETIMEDOUT',\r\n      syscall: 'connect',\r\n      address: '23.23.229.9',\r\n      port: 443\r\n    },\r\n    Error: connect ENETUNREACH 2600:1f18:240c:543b:a8aa:14fb:d008:11fe:443 - Local (:::0)\r\n        at internalConnectMultiple (node:net:1180:16)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -101,\r\n      code: 'ENETUNREACH',\r\n      syscall: 'connect',\r\n      address: '2600:1f18:240c:543b:a8aa:14fb:d008:11fe',\r\n      port: 443\r\n    },\r\n    Error: connect ETIMEDOUT 100.26.116.133:443\r\n        at createConnectionError (node:net:1646:14)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1705:38)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -110,\r\n      code: 'ETIMEDOUT',\r\n      syscall: 'connect',\r\n      address: '100.26.116.133',\r\n      port: 443\r\n    },\r\n    Error: connect ENETUNREACH 2600:1f18:240c:544d:621b:957c:81:5e5b:443 - Local (:::0)\r\n        at internalConnectMultiple (node:net:1180:16)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -101,\r\n      code: 'ENETUNREACH',\r\n      syscall: 'connect',\r\n      address: '2600:1f18:240c:544d:621b:957c:81:5e5b',\r\n      port: 443\r\n    },\r\n    Error: connect ETIMEDOUT 44.199.59.172:443\r\n        at createConnectionError (node:net:1646:14)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1705:38)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -110,\r\n      code: 'ETIMEDOUT',\r\n      syscall: 'connect',\r\n      address: '44.199.59.172',\r\n      port: 443\r\n    },\r\n    Error: connect ENETUNREACH 2600:1f18:240c:5419:4ecf:2a7c:cf52:31ae:443 - Local (:::0)\r\n        at internalConnectMultiple (node:net:1180:16)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -101,\r\n      code: 'ENETUNREACH',\r\n      syscall: 'connect',\r\n      address: '2600:1f18:240c:5419:4ecf:2a7c:cf52:31ae',\r\n      port: 443\r\n    },\r\n    Error: connect ETIMEDOUT 52.20.107.131:443\r\n        at createConnectionError (node:net:1646:14)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1705:38)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -110,\r\n      code: 'ETIMEDOUT',\r\n      syscall: 'connect',\r\n      address: '52.20.107.131',\r\n      port: 443\r\n    },\r\n    Error: connect ENETUNREACH 2600:1f18:240c:540b:c14a:4d57:1251:234d:443 - Local (:::0)\r\n        at internalConnectMultiple (node:net:1180:16)\r\n        at Timeout.internalConnectMultipleTimeout (node:net:1710:5)\r\n        at listOnTimeout (node:internal/timers:575:11)\r\n        at process.processTimers (node:internal/timers:514:7) {\r\n      errno: -101,\r\n      code: 'ENETUNREACH',\r\n      syscall: 'connect',\r\n      address: '2600:1f18:240c:540b:c14a:4d57:1251:234d',\r\n      port: 443\r\n    }\r\n  ]\r\n}\r\n\r\n```\n\n### Additional information\n\n_No response_\n\n### Example\n\n_No response_", "title": "I just converted wsl 2 to wsl 1 and the issue is now solved. It's weird. idk what really happened there. but for some reason wsl 1 do no have any issues."}
